### PR TITLE
Fixup the `install-all` script and AngularJS

### PR DESCRIPTION
**Pull request by robdodson**
Needed to fix an issue where cross-env was causing the build to break.

### CODE REVIEW

**Comment by robdodson** [comment]
I'm not super savvy when it comes to cross-env so i'll have to research if there's a better way to include it in the build command. But I think we can treat that as a separate PR

File: libraries/angular/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "scripts": {
     "test": "cross-env LIBRARY_NAME=@angular/core karma start",
-    "build": "npm run test; cross-env DIST=../../docs/libraries/angular && mkdir -p $DIST && cp -r ./meta $DIST/meta && cp -r ./results $DIST/results"
+    "build": "npm run test; DIST=../../docs/libraries/angular && mkdir -p $DIST && cp -r ./meta $DIST/meta && cp -r ./results $DIST/results"
```
Comment (gkalpak) [inline libraries/angular/package.json]: Ooops, I meant to revert that, but missed it. I was experimenting, trying (unsuccesfully) to get the build to work on Windows.

**Comment by gkalpak** [comment]
Oh, the Travis failures are caused by my moving `karma-MOCHA` into the plugins directory, but failing to update all appropriate paths. Ooops :innocent:

**Comment by robdodson** [comment]
@gkalpak that's what i thought but the paths seem correct... the adapter itself calls require.resolve('mocha') and that seems to be failing but only on travis. trying to work out why that is...

**Comment by robdodson** [comment]
Seems to be working again. Travis has some quirks starting firefox but that's unrelated.

**Comment by gkalpak** [comment]
🎉

**Pull request merged**